### ISSUE TITLE
Workflow Hooks

### ISSUE THREAD

**Issue by anthonyolea** [ovh/celery-director#60]
Handle hooks in the `workflows.yml` file.
Example:
```
demo.EXAMPLE:
  tasks:
    - EXTRACT_TASK
    - TRANSFORM_TASK
    - LOAD_TASK
  hooks:
     success: SUCCESS_TASK
     failure: FAILURE_TASK
```

**Comment by andersdd** [comment]
Absolutely support this idea.  Or even better - support for branching.  If TASK A result = X THEN TASK B ELSE TASK C

**Comment by ncrocfer** [comment]
Fixed by #170
